FIX: Render parameter-less processors as a plain header and the top button row no longer clips when the properties panel is scrolled [UUM-144325] (#2462)
Unity-Technologies/InputSystem: Packages/com.unity.inputsystem/CHANGELOG.md

### Fixed

> - Fixed parameter-less processors and interactions (e.g. "Invert") in the Input Actions editor's Action Properties view showing an expandable foldout whose arrow animated on click but revealed no content; such items are now rendered as a plain, non-collapsible header [UUM-144325](https://issuetracker.unity3d.com/product/unity/issues/guid/UUM-144325)

Reviewer: Can we rephrase this note to appear less robotic?

Reviewer: ```suggestion
- Empty foldouts are no longer shown for processors and interactions that have no settings (e.g. "Invert") [UUM-144325](https://issuetracker.unity3d.com/product/unity/issues/guid/UUM-144325)
```
